IdentityCore/src/oauth2/MSIDExternalSSOContext.m

```diff
@@ -24,7 +24,77 @@
 
 
 #import "MSIDExternalSSOContext.h"
+#import "MSIDWPJKeyPairWithCert.h"
+#import "MSIDKeychainUtil.h"
 
 @implementation MSIDExternalSSOContext
 
+- (MSIDWPJKeyPairWithCert *)wpjKeyPairWithCertWithContext:(id<MSIDRequestContext>)context
+{
+#if TARGET_OS_OSX
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 130000
+    if (@available(macOS 13.0, *))
+    {
+        SecIdentityRef identityRef = [self.loginManager copyIdentityForKeyType:ASAuthorizationProviderExtensionKeyTypeUserDeviceSigning]; // +1
```

Comment:
What is +1 for in the end as comment? Is it for reference counting?

Reply:
yes :)